pkg/3scale/amp/operator/backend_reconciler.go

```diff
@@ -38,7 +41,19 @@ func (r *BackendReconciler) Reconcile() (reconcile.Result, error) {
 	}
 
 	// Listener DC
+	backendRedisSecret := &v1.Secret{}
+	r.Client().Get(r.Context(), types.NamespacedName{
+		Name:      "backend-redis",
+		Namespace: r.apiManager.Namespace,
+	}, backendRedisSecret)
+	redisQueuesUrl := strings.TrimSuffix(string(backendRedisSecret.Data["REDIS_QUEUES_URL"]), "1")
```

Comment:
@valerymo It is expected not to apply the env var on the default case as the default case has logical databases. The redis connection strings on a default case will always be as these are set by the operator
```
redis://backend-redis:6379/1
redis://backend-redis:6379/0
``` 
Sorry there may have been out of date verification steps for the default case if you are expecting the env var to be set. Will review the steps to confirm.

Reply:
Yep out of date steps in the default case will reword them. I was originally setting both env vars to `0` but found that the default install failed if the `LISTENER_WORKER` set. So rather than set them to `0` I removed them.